packages/Python/lldbsuite/test/lang/swift/enum_objc/main.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
```

Comment:
Can you just define the enum in a modularized or bridging header file and import that? Otherwise this depends on the implementation of Foundation and this makes it harder to understand what the test is testing and less fragile.

Reply:
I've never done this, do we have other example in tree?